ssl/src/main/java/org/wildfly/security/ssl/SNIContextMatcher.java

```diff
@@ -60,6 +63,14 @@ public SSLContext getDefaultContext() {
         return defaultContext;
     }
 
+    /**
+     * A class which allows building and configuration SNIContextMatcher. The builder, at minimum requres a default SSLContext.
+     * Other parameters have default values as follows:
+     * <ul>
+     *     <li>The exacts map defaults to being empty</li>
+     *     <li>The wildcards map defaults to being empty</li>
+     * </ul>
```

Comment:
@fazer1929 Just a minor. We don't have to be so specific when it comes to the listing of default values here. It is best to document only things that are not clear. If it was some specific hard coded values list then documenting it would make sense. But here the default is an empty list which can be assumed